docs-web/src/main/java/com/sismics/docs/rest/resource/UserResource.java

```diff
@@ -1076,8 +1076,6 @@ public Response disableTotp(@FormParam("password") String password) {
     @Path("password_lost")
     @Produces(MediaType.APPLICATION_JSON)
     public Response passwordLost(@FormParam("username") String username) {
-        authenticate();
```

Comment:
Does not need authentication?

Reply:
Seems good, this call is useless (but not armful).

Reply:
I don't think a principal is needed here but please check. Since I have mail not activated in my system I cannot check this myself.